runtime/kreivo/src/config/revive/mod.rs

```diff
@@ -0,0 +1,41 @@
+use super::*;
+use pallet_assets::precompiles::{InlineIdConfig, ERC20};
+
+parameter_types! {
+	pub const DepositPerItem: Balance = deposit(1, 0);
+	pub const DepositPerByte: Balance = deposit(0, 1);
+	pub CodeHashLockupDepositPercent: Perbill = Perbill::from_percent(30);
+	pub const ChainId: u64 = 2281;
+}
+
+pub type TrustBackedAssetsInstance = pallet_assets::Instance1;
```

Comment:
Let's call it just KreivoAssets as we don't have other instances and we don't have "TrustBackedAssets"